(function(context) {
    context.ValidatorException = context.Exception;
})(window.Pupil);